apps/ledger-live-desktop/src/renderer/components/OperationsList/EditOperationPanel.tsx

```diff
@@ -1,9 +1,11 @@
+// FIXME: to update when implementing edit transaction on evm
+
```

Comment:
if you want to remove the edit tx feature temporarily. you can remove this file completely or just return null. 
it is not a major issue. It is up to you to keep it here or not.